x-pack/plugins/security/server/authentication/index.ts

```diff
@@ -88,7 +88,7 @@ export async function setupAuthentication({
       authenticationResult = await authenticator.authenticate(request);
     } catch (err) {
       authLogger.error(err);
-      return t.rejected(wrapError(err));
+      return response.internalError(err);
```

Comment:
__question:__ since you got rid of `wrapError` - can we be sure that `internalError` will hide original error message and replace it with something like `An internal server error occurred` for non-Boom errors?

Reply:
don't pass error at all? we have it logged and client shouldn't have access to error details